tools/hangar/src/__snapshots__/invalid.test.ts.snap

```diff
@@ -17,10 +17,11 @@ Error at ../../../../examples/tests/invalid/bring_jsii.w:4:1 | Cannot find modul
 `;
 
 exports[`capture_mutables.w > wing test (--target sim) - invalid > stderr 1`] = `
-"Compilation failed with 5 error(s)
+"Compilation failed with 6 error(s)
 Error at ../../../../examples/tests/invalid/capture_mutables.w:8:10 | Cannot reference 'a' of type 'MutArray<str>' from an inflight context
 Error at ../../../../examples/tests/invalid/capture_mutables.w:10:10 | Cannot reference 's' of type 'MutSet<num>' from an inflight context
 Error at ../../../../examples/tests/invalid/capture_mutables.w:12:10 | Cannot reference 'm' of type 'MutMap<bool>' from an inflight context
+Error at ../../../../examples/tests/invalid/capture_mutables.w:12:20 | Expected type to be \\"num\\", but got \\"(ImmutableMap): num\\" instead
```

Comment:
This doesn't look right, that test is still using `.size`. It should be updated so it's not invalid for the wrong reason 😄

Reply:
In general I would go through all the examples and make sure they're updated, unless it's the intent of the test.
Docs too, in case they're used there. Even though this is temporary we don't want to convey the wrong info.